File: dev-docs/snippets/databases/update-cel-context.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Merge new key-value pairs into a database's CEL context dict.
export async function updateCelContext(client: JsBaoClient, databaseId: string) {
  // #region example
  const db = await client.databases.updateCelContext(databaseId, {
    region: "eu-west",
    tier: "premium",
  });
  // #endregion example
  return db;
}

File: dev-docs/snippets/databases/update-cel-context.swift
import JsBaoClient

// Merge new key-value pairs into a database's CEL context dict.
func updateCelContext(client: JsBaoClient, databaseId: String) async throws {
  // #region example
  let db = try await client.databases.updateCelContext(
    databaseId: databaseId,
    celContext: ["region": "eu-west", "tier": "premium"] as [String: JSONValue]
  )
  // #endregion example
  _ = db
}
